sdk/identity/identity-cache-persistence/samples-dev/persistentCachingInteractiveBrowserCredential.ts

```diff
@@ -0,0 +1,73 @@
+/**
+ * @summary Authenticates using persistent token caching with Interactive Browser Credential
+ */
+import { InteractiveBrowserCredential, useIdentityPlugin, AuthenticationRecord } from "@azure/identity";
+import { cachePersistencePlugin } from "@azure/identity-cache-persistence";
+import path from 'node:path';
+import fs from 'node:fs';
+import { setLogLevel } from "@azure/logger";
+import dotenv from 'dotenv';
+
+useIdentityPlugin(cachePersistencePlugin);
+setLogLevel("verbose");
```

Comment:
nit: should we remove the call to `setLogLevel`? Ideally customers would use AZURE_LOG_LEVEL env var so I wasn't sure if this was intentional

Reply:
done